src/Components/Home.js

```diff
@@ -2,21 +2,37 @@ import './Home.css';
 import { useState } from 'react';
 
 const Home = () => {
-    const [value, setValue] = useState('');
+    const initialValue = [];
+    const [value, setValue] = useState([]);
```

Comment:
the naming convention is confusing and if the value is an array, shouldn't it be plural.